src/components/AcceptTask/TaskDetail.tsx

```diff
@@ -148,7 +157,20 @@ const TaskDetail: FC<ITaskDetail> = ({
           </div>
         )}
       </div>
-
+      <div className={styles['project-description']}>
+        <p> Skills:</p>
+        {selectedTask?.taskSkills?.length > 0 && (
+          <div className={styles['project-attachments']}>
+            {selectedTask?.taskSkills?.map((taskSkills: any, index: number) => (
+              <FileAttachmentCard
```

Comment:
Why are we using `<FileAttachmentCard />` for rendering skills data?

Reply:
Yes